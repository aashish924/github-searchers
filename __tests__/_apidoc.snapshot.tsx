import * as React from 'react';
import renderer from 'react-test-renderer';

import ApiDoc from '../pages/_apidoc'


it('ApiDoc should render correctly', () => {
    const tree = renderer.create(<ApiDoc />).toJSON();
    expect(tree).toMatchSnapshot();
});