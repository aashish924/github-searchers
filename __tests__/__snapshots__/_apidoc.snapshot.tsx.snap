// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApiDoc should render correctly 1`] = `
<div
  className="swagger-ui"
>
  <div
    className="loading-container"
  >
    <div
      className="info"
    >
      <div
        className="loading-container"
      >
        <div
          className="loading"
        />
      </div>
    </div>
  </div>
</div>
`;
